models/fixtures/team.yml

```diff
@@ -37,6 +37,7 @@
   num_repos: 0
   num_members: 1
   unit_types: '[1,2,3,4,5,6,7]'
+
```

Comment:
nit: please remove

Reply:
There was missing 1 empty line, now it is consistent.

Reply:
You are right, my mistake